clients/js/src/ChromaClient.ts

```diff
@@ -2,7 +2,6 @@ import { AdminClient } from "./AdminClient";
 import { authOptionsToAuthProvider, ClientAuthProvider } from "./auth";
```

Comment:
Please have @codetheweb review JS changes

Reply:
(@spikechroma lmk when https://github.com/chroma-core/chroma/pull/2714 lands and you rebase on that, I'll review then)

Reply:
sg